addons/hr/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,34 @@
+---Fields in module 'hr'---
+hr           / hr.department            / message_last_post (datetime)  : DEL 
+hr           / hr.department            / message_main_attachment_id (many2one): NEW relation: ir.attachment
+# NOTHING TO DO
+
+hr           / hr.employee              / activity_ids (one2many)       : NEW relation: mail.activity
+hr           / hr.employee              / additional_note (text)        : NEW 
+hr           / hr.employee              / certificate (selection)       : NEW selection_keys: ['bachelor', 'master', 'other']
+hr           / hr.employee              / children (integer)            : previously in module hr_contract
```

Comment:
@mreficent I think that you have move children and place_of_brith from hr_contract to hr.
https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade.py#L1921

Reply:
@sergio-teruel This is not needed in this case, as `hr` is loaded before than `hr_contract`, so the XML-ID for `hr` will be created first, and thus no removal will be done.